isos/bootstrap-staging.sh

```diff
@@ -106,6 +106,7 @@ yum_cached -c $cache -u -p $PKGDIR install \
     libtirpc \
     grep \
     haveged \
+    dbus \
```

Comment:
Curious.  We can't just echo hostname > /etc/hostname  and set /etc/hosts appropriately?